src/jdk.jartool/share/classes/sun/tools/jar/resources/jar.properties

```diff
@@ -307,6 +309,12 @@ main.help.opt.create.update.index.date=\
 \      --date=TIMESTAMP       The timestamp in ISO-8601 extended offset date-time with\n\
 \                             optional time-zone format, to use for the timestamps of\n\
 \                             entries, e.g. "2022-02-12T12:30:00-05:00"
+main.help.opt.extract=\
+\ Operation modifiers valid only in extract mode:\n
+main.help.opt.extract.keep-old-files=\
+\  -k, --keep-old-files       Do not overwrite existing files.\n\
+\                             In particular, if a file appears more than once in an\n\
```

Comment:
In addition, we need to update the help to clarify -x. --extract behavior that it will overwrite by default.

Here is the verbiage from tar to give you a start

` -x      Extract to disk from the archive.  If a file with the same name appears more than
             once in the archive, each copy will be extracted, with later copies overwriting
             (replacing) earlier copies.  The long option form is --extract.`

Reply:
Updated.

Reply:
Hello Henry, I think this `-k` option help text would need a slight modification. Right now it states that if a file appears more than once in an archive, then this setting this flag will not overwrite the earlier copies. In reality, it doesn't matter how many times the file appears in the archive - even if it appears just once, and if the file is being extracted into a directory which already has the same named file at that path, then this `-k` flag will not overwrite that existing file.

So I think we should reword it to talk about its behaviour in context of some file already existing in the destination directory where the jar contents are being extracted. 

I wonder if we should make a mention that we don't do any case sensitive checks for the existence of the file in the destination directory and instead we use filesystem API to check for existence (which depending on the filesystem may be case insensitive, like macosx).

Reply:
You have valid point. I refer to the tar man page, and I thought do not overwrite existing files says it. The in particular is just to call out a specific use case.

Reply:
Please review the update and see if that's good enough.